docs/contributing/index.md

```diff
@@ -43,6 +43,28 @@ Once you've completed these steps, you are ready to submit your contribution.
 
 Once you've completed these steps, you are ready to submit your contribution.
 
+
+### Contributing to Documentation
+
+#### Documentation Style
+
+To ensure that our code is well-documented and easy to understand, we use [Google-style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) to document all functions, classes, and methods in this library.
+
+#### Getting Help with Setup and Configuration
+
+To work on documentation locally, follow these steps:
+
+- To setup the documentation, navigate to the scripts directory
+- Run the `scripts/build-docs.sh` script to build the documentation
+- Serve the documentation locally, using the `scripts/docs-live.sh` script
```

Comment:
``````suggestion
To work on documentation locally, we provide a script that will automatically re-render the docs when you make changes:

```
./scripts/docs-live.sh
```
``````